Dockerfile

```
@@ -22,6 +22,13 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+FROM python:slim
```

Named stages make it a little clearer
```suggestion
FROM python:slim AS src
```